docs/quick-start.md

```diff
@@ -0,0 +1,188 @@
+- [vcluster CLI](https://www.vcluster.com/docs/getting-started/setup)
+- [curl](https://curl.se/)
+
+Ensure that `clusterctl` has the `vcluster` provider repository configured.
+```shell
+clusterctl config repositories | grep vcluster
+vcluster       InfrastructureProvider   https://github.com/loft-sh/cluster-api-provider-vcluster/releases/latest/                 infrastructure-components.yaml
+```
+If the output is empty, please upgrade your `clusterctl` version.
+
+## Instructions
+Using `kind` we will create a management cluster for Cluster API to use.
+```shell 
+kind create cluster
+
+Creating cluster "kind" ...
+ ✓ Ensuring node image (kindest/node:v1.23.4) 🖼
+ ✓ Preparing nodes 📦  
+ ✓ Writing configuration 📜 
+ ✓ Starting control-plane 🕹️ 
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-kind"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-kind
+
+Thanks for using kind! 😊
+```
+
+Verify that `kubectl` has been correctly configured by running.
+
+```shell
+# Checking the current context
+kubectl config current-context 
+kind-kind
+
+# Getting the namespaces in the new cluster
+kubectl get namespace
+NAME                 STATUS   AGE
+default              Active   110s
+kube-node-lease      Active   111s
+kube-public          Active   111s
+kube-system          Active   111s
+local-path-storage   Active   108s
+```
+
+Install the `vcluster` provider in the cluster.
+```shell
+clusterctl init --infrastructure vcluster
+```
+
+Next we will create our virtual cluster within our `kind` cluster.
+```shell
+export CLUSTER_NAME=kind
+export CLUSTER_NAMESPACE=vcluster
+# Note same version as kind cluster above
+export KUBERNETES_VERSION=1.23.4 
+export HELM_VALUES=""
+
+kubectl create namespace ${CLUSTER_NAMESPACE}
+clusterctl generate cluster ${CLUSTER_NAME} \
+    --infrastructure vcluster \
+    --kubernetes-version ${KUBERNETES_VERSION} \
+    --target-namespace ${CLUSTER_NAMESPACE} | kubectl apply -f -
+```
+
+We can verify the cluster by checking the host clusters namespaces.
+```shell
+kubectl get namespace
+NAME                                   STATUS   AGE
+capi-kubeadm-bootstrap-system          Active   2m12s
+capi-kubeadm-control-plane-system      Active   2m12s
+capi-system                            Active   2m12s
+cert-manager                           Active   2m23s
+cluster-api-provider-vcluster-system   Active   2m11s
+default                                Active   7m4s
+kube-node-lease                        Active   7m5s
+kube-public                            Active   7m5s
+kube-system                            Active   7m5s
+local-path-storage                     Active   7m2s
+vcluster                               Active   48s
+```
+
+`vcluster` is the namespace for our virtual cluster. Now lets attempt to use it.
+
+**Note! Keep the following command running for the duration of this tutorial**
+```shell
+vcluster connect kind -n vcluster
```

Comment:
A tip: if you were to create the vcluster with the NodePort service, the connect command would set up kube config without port-forwarding and exit. This only works on some local clusters, but it should work with kind. The required helm values:
```
service:
  type: NodePort
```